devicon.json

```diff
@@ -5569,8 +6150,34 @@
             }
         ]
     },
+    {
+        "name": "storybook",
+        "altnames": [
+            "storybookjs"
+        ],
+        "tags": [
+            "framework",
+            "documentation", 
+            "ui"
+        ],
+        "versions": {
+          "svg": [
+              "original", 
+              "original-wordmark",
+              "plain", 
+              "plain-wordmark"
+          ],
+          "font": [
+              "plain", 
+              "plain-wordmark"
+          ]
+        },
+        "color": "#FF4785",
+        "aliases": []
+    },
```

Comment:
Duplicate entry for `storybook`

Reply:
3c5bf839850823623296b0f0a222ba65aff6386c